app/Http/Controllers/Multiplayer/RoomsController.php

```diff
@@ -46,7 +46,15 @@ public function index()
         }
 
         $search = Room::search($params);
-        $rooms = $search['query']
+        $query = $search['query'];
+
+        // temporary workaround for lazer client failing to deserialise `daily_challenge` room category
+        // can be removed 20241129
+        if ($apiVersion < 20240529) {
```

Comment:
Not sure if you do this sort of thing around these parts, but I don't foresee this living too long, so I did the standard that we do in client of only keeping this around for ~6 months at which point it can be nuked.

Reply:
maybe note that it's temporary in the docs?